swagger.yaml

```diff
@@ -183,8 +185,12 @@ paths:
           type: string
         - name: source_metadata
           in: query
-          description: '???'
+          description: >-
+            A JSON blob containing extra source specific information.  This
+            field is free-form, but we generally expect the data will follow
+            one of the formats outlined in the `/schema` folder.
```

Comment:
Are you sure that URL works?  My attempts to load existing assets seem to fail:
- https://api.monitoring.envirodatagov.org/assets/swagger.yml (404 & bytes)
- https://api.monitoring.envirodatagov.org/assets/home.js (404 & html)